### ISSUE TITLE
Add your NAME to Contributors.md

### ISSUE THREAD

**Issue by udit-001** [udit-001/Collaboration-For-Beginners#2]
Add your name to the Contributors.md file using the below convention:

```
#### Name: [YOUR NAME](GitHub Link)
- Place: City, State, Country
- Bio: Who are you?
- GitHub: [Your Name](GitHub Link)
```

**Comment by kallepallimounikasmitha** [comment]
Can I add my name in Contributors.md?

**Comment by udit-001** [comment]
@kallepallimounikasmitha Yes, sure! Just follow the instructions in README.md and create a pull request.

**Comment by bullywhippet** [comment]
Hey thanks for making this repo and letting me contribute!

**Comment by davidsjk** [comment]
This is a great idea! and it's been going on for 2 years! Awesome.
Thank you for doing this.

**Comment by kmryashasvi** [comment]
> Can I add my name in Contributors.md?

yes, of course

**Comment by MIT-8** [comment]
Well, I'm a first-timer. 
And, My pull request number is #212.

**Comment by msultont** [comment]
hi udit-001! I just created a pull request and this is the [LINK](https://github.com/udit-001/Collaboration-For-Beginners/pull/267) of it! Proceeding to merge to the master branch would be awesome! Thanks anyway
